projects/plugins/protect/src/class-threats.php

```diff
@@ -144,6 +144,10 @@ public static function fix_threats( $threat_ids ) {
 	 * @return bool|array
 	 */
 	public static function fix_threats_status( $threat_ids ) {
+		if ( ! $threat_ids ) {
```

Comment:
```suggestion
		if ( empty( $threat_ids ) ) {
```
Suggestion, could also return early for empty arrays.